roles/jupyterhub/server/templates/jupyterhub_config.py.j2

```diff
@@ -130,3 +130,18 @@ def pre_spawn_hook(spawner):
 {% for key, value in jupyterhub_properties.get('yarnspawner', {}).items() +%}
 c.YarnSpawner.{{ key }} = {% if value is string and not value.startswith('f:') %}'{{ value }}'{% else %}{{ value if value is not string else value.lstrip('f:') }}{% endif %}
 {% endfor %}
+

```

Comment:
Perharps here needs few line breaks to separate logging configuration section.

Reply:
i kept the same existing config, but, we can set `INFO` as Hue

Reply:
thanks !